### DISCUSSION TITLE
Windows在线打包不支持中文名 name_zh不起作用  标题和任务栏不显示文字只有图标

### DISCUSSION THREAD

**Discussion by iomect** [tw93/Pake#611] [Q&A]
使用中文name在线打包就会提示这个报错  
`No files were found with the provided path: node_modules/pake-cli/output/*. No artifacts will be uploaded.`

我以为是action中cli打包方式无法配置 name_zh 导致的
然后把我需要的添加到了 "Pake/master/apps.conf.json" 中
然后使用"Build Single Popular App"来打包
但name_zh参数好像并没有起到什么作用, 安装过程中的提示依然是英文名, 安装完成后自动创建的桌面快捷方式也是英文名, 并且在任务栏中名字是空的.
(搜过issue和discuss 改用英文可以解决 但期望可以支持中文)

按照大家的习惯, 针对国内用户的话, 最好还是在安装的过程中就显示中文
"Welcome to the 中文名字 Setup Wizard"  并且安装完成后的桌面快捷方式也是中文名
然后标题为空虽然是设定,  但最好是可以自定义标题和任务栏上的文字...
希望下一步可以更新这些支持~

**Comment by iomect** [comment]
另外就是不知道能不能在win下直接生成绿色版单文件, 或者是绿色版打开后自动向AppData写入需要的文件.

**Comment by Tlntin** [comment]
tauri那边不支持中文名，改起来会比较麻烦

**Comment by kewillcai** [comment]
我也遇到这个问题，名称用中文就不行
